bears/general/SpaceConsistencyBear.py

```diff
@@ -1,22 +1,40 @@
-from coalib.bearlib import deprecate_settings
 from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
+from coalib.bearlib.aspects import map_setting_to_aspect
+from coalib.bearlib.aspects.Formatting import (
+    Formatting,
+    TrailingSpace,
+    Indentation,
+    NewlineAtEOF
+)
 
 
-class SpaceConsistencyBear(LocalBear):
+class SpaceConsistencyBear(
+        LocalBear,
+        aspects={
+            'detect': [
+                Indentation,
+                NewlineAtEOF,
+                TrailingSpace
+            ]},
+        languages=['Python']
+        ):
     LANGUAGES = {'All'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
 
-    @deprecate_settings(indent_size='tab_width')
+    @map_setting_to_aspect(
+        enforce_newline_at_EOF=NewlineAtEOF,
+        allow_trailing_whitespace=TrailingSpace,
+        indent_size=Indentation.indent_size)
     def run(self,
             filename,
             file,
-            use_spaces: bool,
+            use_spaces: bool=True,
```

Comment:
If you set a default here, you'll have to adjust `test_needed_settings` to reflect the new setting count.